SOURCES/etc/xapi.d/plugins/xcpngutils/__init__.py

```diff
@@ -63,12 +63,24 @@ def log_unhandled_exception(origin, exception_type, exception_value,
     return logger
 
 
+class ProcessException(Exception):
+    def __init__(self, code, command, stderr, stdout) -> None:
```

Comment:
I can understand the type hint for methods, but why for constructor?

Reply:
Auto completion added it.